starknet_devnet/udc.py

```diff
@@ -0,0 +1,58 @@
+"""UDC and its constants"""
+
+from starkware.python.utils import to_bytes
+from starkware.solidity.utils import load_nearby_contract
+from starkware.starknet.services.api.contract_class import ContractClass
+from starkware.starknet.testing.contract import StarknetContract
+from starkware.starknet.testing.starknet import Starknet
+
+
+class UDC:
+    """UDC wrapper class"""
+
+    CONTRACT_CLASS: ContractClass = None  # loaded lazily
+
+    # Precalculated
+    # HASH = to_bytes(compute_class_hash(contract_class=UDC.get_contract_class()))
+    HASH = 3484004124736344420122298338254154090450773688458934993781007232228755339881
```

Comment:
I assume that currently it was calculated based on `to_bytes(compute_class_hash(contract_class=UDC.get_contract_class()))` and it's working but how about the future when for example `compute_class_hash` will change? 

I'm not sure if there is a possibility of that. I just wrote down my concern.

Same with `ADDRESS = calculate_contract_address_from_hash(salt=10, class_hash=HASH` above.

Reply:
discussed on the call